### PR TITLE
Feature/global renaming to patternatlas

### PR DESCRIPTION
**Pull request by manuwei**
* rename all occurence of the old PatternPedia name to PatternAtlas
* rename group name to follow naming conventions of the Quantil orga (resolves [issue-description](https://github.com/orgs/UST-QuAntiL/projects/2#card-63295966)) 
* change version

depends on [ui-PR](https://github.com/PatternAtlas/pattern-atlas-ui/pull/110)

### CODE REVIEW

File: intellij-idea-code-style.xml [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-            <package name="java" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="com.patternpedia.api" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="" withSubpackages="true" static="true" />
+            <package name="java" withSubpackages="true" static="false"/>
+            <emptyLine/>
+            <package name="io.github.ust.quantil.patternatlas.api" withSubpackages="true" static="false"/>
```
Comment (lharzenetter) [inline intellij-idea-code-style.xml:11]: Wouldn't this be `io.github.patternatlas.api`? (as https://patternatlas.github.io/pattern-atlas-api/)

File: pom.xml [hunk 2]
```diff
@@ -9,9 +9,9 @@
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
 
-    <groupId>com.patternpedia</groupId>
-    <artifactId>api</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <groupId>io.github.ust.quantil.patternatlas</groupId>
```
Comment (lharzenetter) [inline pom.xml:12]: See above

**Review by lharzenetter — CHANGES_REQUESTED** [review]

File: intellij-idea-code-style.xml (hunk 1 above)
Comment (lharzenetter) [reply intellij-idea-code-style.xml:11]: Somehow this didn't get updated...

File: src/main/java/io/github/patternatlas/api/entities/candidate/Candidate.java [hunk 3]
```diff
@@ -30,7 +36,7 @@
     @ManyToOne
     private PatternLanguage patternLanguage;
 
-//    @Type(type = "jsonb")
+    //    @Type(type = "jsonb")
```
Comment (lharzenetter) [inline src/main/java/io/github/patternatlas/api/entities/candidate/Candidate.java:39]: Maybe delete these commented lines?

File: src/main/java/io/github/patternatlas/api/entities/user/UserEntity.java [hunk 4]
```diff
@@ -60,15 +82,16 @@
     @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
     private List<CandidateComment> candidateComments = new ArrayList<>();
 
-    /** Pattern fields*/
+    /**
+     * Pattern fields
+     */
 //    @JsonIgnore
 //    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
```
Comment (lharzenetter) [inline src/main/java/io/github/patternatlas/api/entities/user/UserEntity.java:89]: Maybe delete these commented lines?

**Review by lharzenetter — CHANGES_REQUESTED** [review]

**Comment by manuwei** [comment]
I fix the last instance where the package name need to be corrected. I am aware of that there is too much commented code for the pattern candidates/usermangement feature - but I think that's a separate bug/feature to be considered, so I added an [issue](https://github.com/PatternAtlas/pattern-atlas-api/issues/28) for it.

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**